plugin/pkg/shoot/validator/admission.go

```diff
@@ -843,6 +843,49 @@ func (c *validationContext) validateAdmissionPlugins(a admission.Attributes, sec
 	return allErrs
 }
 
+// For backwards-compatibility, we want to validate the scaleDownUnneededTime and scaleDownDelayAfterAdd only for newly created Shoot clusters.
+// Performing the validation for all Shoots would prevent already existing Shoots with the wrong spec to be updated/deleted.
+func (c *validationContext) validateScaleDownConfiguration(a admission.Attributes) field.ErrorList {
+	var (
+		allErrs                field.ErrorList
+		stringRepresentation   string
+		path                   = field.NewPath("spec", "kubernetes", "clusterAutoscaler")
+		scaleDownUnneededTime  = 30 * time.Minute //default
+		scaleDownDelayAfterAdd = 1 * time.Hour    //default
+	)
+
+	if a.GetOperation() != admission.Create {
+		return nil
+	}
+
+	//in order to check in the case where the ScaleDownDelayAfterAdd is set generally, but the worker settings may set and overwrite ScaleDownUnneededTime
+	if c.shoot.Spec.Kubernetes.ClusterAutoscaler != nil {
+		if c.shoot.Spec.Kubernetes.ClusterAutoscaler.ScaleDownDelayAfterAdd != nil {
+			scaleDownDelayAfterAdd = c.shoot.Spec.Kubernetes.ClusterAutoscaler.ScaleDownDelayAfterAdd.Duration
+		}
+		if c.shoot.Spec.Kubernetes.ClusterAutoscaler.ScaleDownUnneededTime != nil {
+			scaleDownUnneededTime = c.shoot.Spec.Kubernetes.ClusterAutoscaler.ScaleDownUnneededTime.Duration
+		}
+	}
+
+	path = field.NewPath("spec", "provider", "workers", "clusterAutoscaler")
+	for _, worker := range c.shoot.Spec.Provider.Workers {
+		if worker.ClusterAutoscaler == nil || worker.ClusterAutoscaler.ScaleDownUnneededTime == nil {
+			continue
+		} else if worker.ClusterAutoscaler.ScaleDownUnneededTime.Duration < 1*time.Minute && scaleDownDelayAfterAdd < 1*time.Minute {
+			stringRepresentation = "ScaleDownDelayAfterAdd: " + scaleDownDelayAfterAdd.String() + ", ScaleDownUnneededTime: " + worker.ClusterAutoscaler.ScaleDownUnneededTime.Duration.String()
+			allErrs = append(allErrs, field.Invalid(path.Child("scaleDownUnneededTime"), stringRepresentation, "ScaleDownUnneededTime and ScaleDownDelayAfterAdd should not be less than a minute at the same time"))
+		}
+	}
+
+	if scaleDownDelayAfterAdd < 1*time.Minute && scaleDownUnneededTime < 1*time.Minute {
```

Comment:
What if all workers override `ScaleDownUnneededTime`? Example scenario:
When
```
				worker2 := core.Worker{
					...
					ClusterAutoscaler: &core.ClusterAutoscalerOptions{
						ScaleDownUnneededTime: workerUnneededTime,
					},
				}
```
The following case fails but should pass
```
Entry("should allow when configuration for both workers ScaleDownUnneededTime is > 1m, ScaleDownUnneededTime is < 1m and ScaleDownDelayAfterAdd is < 1m", ptr.To(metav1.Duration{Duration: 1 * time.Hour}), ptr.To(metav1.Duration{Duration: 1 * time.Second}), ptr.To(metav1.Duration{Duration: 1 * time.Second}), BeNil()),
```